digitalfabrik/entitlementcard
# 1343: Add english translations to store descriptions

### Short description

Since the proposal for store translations was not accepted by nuernberg, the english translations will be simply added with line breaks to the existing store description

### Proposed changes



- add additional column for csv import
- merge discounts for acceptance store
- split discounts for csv export
- adjusted documentation since we will get csv files instead excel sheets from nuernberg

### Side effects



- Not very happy with that implementation but since nuernberg doesn't want to pay for proper translation implementation, thats the simplest way to display them

### Resolved issues



Fixes: #1343 

### Notes

- open pr of the data repository can be found here:
https://github.com/digitalfabrik/sozialpass-nuernberg-data/pull/11

### Testing
- run "Import data" from backend runConfigs (dev only)
- check store descriptions in app

![image](https://github.com/digitalfabrik/entitlementcard/assets/37902063/c47e292c-7770-40aa-84ba-c9d88af8a9f8)
